app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
+
+> Why should I care?
+
+You could use help
```

Comment:
> People aren't contributing (or not in the ways you want)

I think this is the key thing. I was headed in the direction of "spread the burden / avoid burnout" but both of those are very negative, whereas getting more and more useful contributions is positive.